### DISCUSSION TITLE
Redirect to an arbitrary URL

### DISCUSSION THREAD

**Discussion by esisl** [pimcore/pimcore#9737] [Q&A / Support]
Can I redirect from a controller to an arbitrary url?
$this->redirect($url);    Doesn't work, the page throws error 500 and that's it :(

**Comment by TS1997** [comment]
@esisl `return $this->redirect('http://google.se');` should work. Did you return the call? Can you add details about the 500-error you're getting if you did return the call? If not, add return in front of the call. Another thing to note is that the url has to contain the protocol, i.e. http:// or https://.

**Comment by esisl** [comment]
Sorry!
I typed the answer and did not send it - I really just had a typo. The head fell off. Corrected - it worked.
